Runtime/CesiumCameraController.cs

```diff
@@ -449,11 +456,39 @@ private bool RaycastAlongForwardVector(float raycastDistance, out float hitDista
         private void HandlePlayerInputs()
         {
             #if ENABLE_INPUT_SYSTEM
-            Vector2 lookDelta = lookAction.ReadValue<Vector2>();
+            Vector2 lookDelta;
+            Vector2 moveDelta;
+            lookDelta = lookAction.ReadValue<Vector2>();
+            moveDelta = moveAction.ReadValue<Vector2>();
+
+#if UNITY_IOS || UNITY_ANDROID
+            bool handledMove = false;
+            bool handledLook = false;
+
+            foreach(var touch in Touch.Touch.activeTouches)
+            {
+                if(touch.startScreenPosition.x < Screen.width / 2)
```

Comment:
I don't have the ability to test this on my own, so do you mind clarifying what the controls are? I see that you can move by dragging on the left hand side of the screen, but what does `touch.delta` mean?

Reply:
Also I assume this is without some sort of UI, right? So the user just drags on the screen, on top of their scene?

Reply:
Yep, usually you play mobile games with thumbs in landscape mode. the left thumb moves the player, and the right thumb moves around. There is usually a circle with a dot inside that move like thumb-sticks. Here there is no UI, however I don't think its necessary to show this.

The left thumb doesn't use delta, so that the camera moves proportional to the amount it moved from the starting position. This way the camera still moves even without moving the thumb.

The right thumb uses delta so that the camera only looks around when the thumb moves. The sensitivity for both thumbs works out perfectly so it doesn't need to be scaled.

Reply:
That makes sense, thanks for explaining :pray: